bioconductor/templates/_helpers.tpl

```diff
@@ -89,9 +89,10 @@ Creates the bash command for the init containers used to place files and change
 */}}
 {{- define "bioconductor.init-container-commands" -}}
 cp -anrL /opt/configs/readonly/rstudio/ /home/;
-chown -R rstudio:rstudio /home/rstudio;
 {{- if and .Values.libraries.persistence.enabled (not .Values.libraries.persistence.separateClaim.enabled) }}
 mkdir -p {{.Values.persistence.mountPath}}/persisted-library/R;
+chown -R rstudio:rstudio {{.Values.persistence.mountPath}}/persisted-library/R;
```

Comment:
Is it possible to use fsGroup here instead? That same fix was made in galaxy-helm and works well.

Reply:
I think it will work with `fsgroup`. I can try that

Reply:
No, it will mount the volume with different group permissions. Something like `root:rstudio` when you `ls -lh` so ownership won't change but `rstudio` user will have read access. This will work better for read-only scenario I believe